src/RazorSdk/SourceGenerators/StaticTagHelperFeature.cs

```diff
@@ -6,12 +6,20 @@
 
 namespace Microsoft.NET.Sdk.Razor.SourceGenerators
 {
-    internal sealed class StaticTagHelperFeature : ITagHelperFeature
+    internal sealed class StaticTagHelperFeature : RazorEngineFeatureBase, ITagHelperFeature
```

Comment:
Why do we need changes to this type?

Reply:
It's not strictly necessary but `RazorEngineFeatureBase` already contains a null-handling getter/setter for the `Engine` type ([ref](https://github.com/dotnet/aspnetcore/blob/main/src/Razor/Microsoft.AspNetCore.Razor.Language/src/RazorEngineFeatureBase.cs#L13-L26)) so I figured we can reuse that here.

Also it seems like the intention was for `RazorEngineFeatureBase` to be inherited in this way so figured I'd follow the best practice.